src/Alpha_complex/utilities/alpha_complex_persistence.cpp

```diff
@@ -61,10 +63,12 @@ std::vector<double> read_weight_file(const std::string &weight_file) {
 
 template<class Kernel>
 Simplex_tree create_simplex_tree(const std::string &off_file_points, const std::string &weight_file,
-                                 bool exact_version, Filtration_value alpha_square_max_value) {
+                                 bool exact_version, Filtration_value alpha_square_max_value,
+                                 bool output_squared_root_values) {
```

Comment:
Again for `output_squared_root_values`.

Reply:
Modified by [44a6747](https://github.com/GUDHI/gudhi-devel/pull/1172/commits/44a6747c1c2ffd0d80db14bfd65d64cd9f0e6e88), [4227322](https://github.com/GUDHI/gudhi-devel/pull/1172/commits/4227322a044ffd87c2a0d3743d8bd9a6f1d66bf1) and [404cb39](https://github.com/GUDHI/gudhi-devel/pull/1172/commits/404cb3951e1c9f6283a2ccc3c39ce340b65f9ddd)